packages/tests/backend/sirius-components-graphql-tests/src/main/java/org/eclipse/sirius/components/graphql/tests/ExecuteEditingContextFunctionEventHandler.java

```diff
@@ -57,7 +57,7 @@ public void handle(Sinks.One<IPayload> payloadSink, Sinks.Many<ChangeDescription
 
             try {
                 payload = executeEditingContextFunctionInput.function().apply(editingContext, input);
-            } catch (Exception exception) {
+            } catch (Exception | AssertionError exception) {
```

Comment:
This allows to fail fast when using AssertJ in `ExecuteEditingContextFunctionInput`. Otherwise the handler times out, and the error doesn't help understand the issue.